script/ci-build-libsass

```diff
@@ -1,89 +0,0 @@
-#!/bin/bash
```

Comment:
What's the justification for moving this file? This change would appear to be inconsequential to this PR.